decentraland/screens/MyAccountScreen.js

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import MyAccount from "@presentational/MyAccount";
+import { connect } from "react-redux";
+import AccountCreationActions from "@constants/AccountCreationActions";
+import ActionStatus from "../constants/ActionStatus";
+import PropTypes from "prop-types";
+
+export class MyAccountScreen extends React.Component {
+  static navigationOptions = {
+    title: "My Account",
+  };
+
+  render() {
+    const { accountInfo } = this.props;
+    const creationActions = [];
+    creationActions.push({
+      name: "Account created",
+      action: null,
+      status: !accountInfo.account ? ActionStatus.MISSING : ActionStatus.DONE,
+    });
+    Object.keys(AccountCreationActions).forEach(action => {
+      const creationAction = {
+        name: AccountCreationActions[action].name,
+        action,
+      };
+      if (accountInfo.creationActions.hasOwnProperty(action)) {
+        creationAction.status = ActionStatus.DONE;
+      } else {
+        creationAction.status = ActionStatus.MISSING;
+      }
+      creationActions.push(creationAction);
+    });
+    return (
+      <MyAccount
+        progress={this._getPercentage(!!accountInfo.account, creationActions)}
```

Comment:
Maybe extract the `!!accountInfo.account` to a boolean named `isAccountCreated` and then use that here to ensure the reader knows what this is for without looking ahead to the parameters of the `_getPercentage` function